spring-kafka/src/main/java/org/springframework/kafka/annotation/KafkaListenerAnnotationBeanPostProcessor.java

```diff
@@ -664,7 +665,17 @@ private MessageHandlerMethodFactory createDefaultMessageHandlerMethodFactory() {
 					(KafkaListenerAnnotationBeanPostProcessor.this.beanFactory instanceof ConfigurableBeanFactory ?
 							(ConfigurableBeanFactory) KafkaListenerAnnotationBeanPostProcessor.this.beanFactory : null);
 
-			DefaultFormattingConversionService conversionService = new DefaultFormattingConversionService();
+			ConversionService conversionService;
+			try {
+				conversionService =
+						KafkaListenerAnnotationBeanPostProcessor.this.beanFactory.getBean(ConversionService.class);
+			}
+			catch (Exception e) {
+				KafkaListenerAnnotationBeanPostProcessor.this.logger.warn(
+						"Failed to get bean for type ConversionService. "
+								+ "Using DefaultFormattingConversionService.", e);
```

Comment:
I think we don't need this log message at all. At least on the `info` (or even `debug`) level...
And would be better to say:

> No ConversionService bean in the application context. Using DefaultFormattingConversionService as default instance.